src/gazebo_gps_plugin.cpp

```diff
@@ -86,34 +86,34 @@ void GpsPlugin::Load(physics::ModelPtr _model, sdf::ElementPtr _sdf)
 
   if (env_lat) {
     lat_home = std::stod(env_lat) * M_PI / 180.0;
+    gzmsg << "Home latitude is set to " << lat_home << ".\n";
```

Comment:
Why was this moved here?

Reply:
Because I wouldn't print the lat/lon if it's just the default.

Reply:
If the world_has_origin is true, it still is not default. Could we add another print out there?

Reply:
Ah ok, done in 314de47.